fastlane/Fastfile

```diff
@@ -477,13 +477,25 @@ platform :ios do
       download_dir: parent_dir_for_lprojs
     )
 
-    # Then redispatch the appropriate subset of translations back to the `InfoPlist.strings` and `Sites.strings` files in corresponding `*.lproj` dirs
-    parent_dir_for_intents_target_lprojs = File.join('WordPress', 'WordPressIntents')
```

Comment:
My editor told me about this unused variable 😄 

Notice that the `WordPressIntents` app extension which the variable referred to is part of the `MANUALLY_MAINTAINED_STRINGS_FILES` `Hash` which we pass to current iteration of the extractor action.

Reply:
This was already fixed by my "cleanup Fastfile" PR (which is build on top of my "split Fastfile" PR)… so committing this change here will likely lead to conflicts in both PR 🫤